src/routes/_components/IconButton.html

```diff
@@ -52,27 +52,18 @@
    * regular styles
    */
 
-  :global(.icon-button:hover .icon-button-svg) {
-    fill: var(--action-button-fill-color-hover);
+  .icon-button:hover {
+    background: var(--action-button-bg-color-hover);
   }
 
-  :global(.icon-button.not-pressable:active .icon-button-svg,
-  .icon-button.same-pressed:active .icon-button-svg) {
-    fill: var(--action-button-fill-color-active);
+  .icon-button:active {
+    background: var(--action-button-bg-color-active);
   }
```

Comment:
IIUC the goal here is to use background colors rather than the colors of the SVG buttons to communicate hover vs active states?

Personally I liked the subtlety of the previous design here, especially for the boost/fav icons in the status toolbar. I think it meshes well with the animation when you press it too. Only one thing changes in the process of hover-click-unclick: the color of the icon.

Given that we now have the checkmark for the boost/fav icons, is this a must-have?